CONTRIBUTING.md

```diff
@@ -4,18 +4,18 @@ We love pull requests! Our process for accepting changes has a few steps:
 
 1. If you haven't submitted anything before, and you aren't (yet!) a member of our organization, fork and clone the repo:
```

Comment:
I just merged it in [here](https://github.com/tomeshnet/documents/blob/master/CONTRIBUTING.md). 
The link can be updated and contrib removed... thought I didn't commit either when I did the first pass, looks like they go swept up at some point.